scripts/install_odb_2.5.sh

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
+trap cleanup EXIT
+
+declare ODB_INSTALL_DIR="/usr/local"
+declare ODB_BUILD_DIR="/tmp/odb"
+declare success="false"
+
+function cleanup() {
+    echo "Cleaning up Odb temporaries."
+
+    rm --recursive --force "/tmp/build2src"
+    
+    if [[ -n "${ODB_BUILD_DIR}" ]]; then
+        rm --recursive --force "${ODB_BUILD_DIR}"
+    fi
+    
+    if [[ "${success}" != "true" ]]; then
+        if [[ -d "${ODB_INSTALL_DIR}" ]]; then
+            rm --recursive --force "${ODB_INSTALL_DIR}"
+        fi
+    fi
+}
+
+function usage() {
+    cat <<EOF
+This is a convenience script that installs the build2 build system and
+uses that to build and install the odb compiler, runtime library and also
+the slite and pgsql support library.
+
+${0} [-h] [-d] <odb_install_path> 
+  -h  Print this usage information, and quit.
+  -d  Install directory of odb. Optional. /usr/local is the deafault.
+EOF
+}
+
+while getopts "hd:" OPTION; do
+    case ${OPTION} in
+        h)
+            usage
+            exit 0
+            ;;
+        d)
+            ODB_INSTALL_DIR="${OPTARG}"
+            ;;
+        *)
+            usage >&2
+            exit 1
+            ;;
+    esac
+done
+
+
+#Build2 toolchain
+mkdir /tmp/build2src
+pushd "/tmp/build2src"
```

Comment:
To make sure we do not clash with anything on the system even if running outside of docker, use `mktemp -d` with an optional suffix or something.